source/adapters/level_zero/command_buffer.cpp

```diff
@@ -1754,7 +1757,7 @@ ur_result_t updateKernelCommand(
   auto Platform = CommandBuffer->Context->getPlatform();
   ZE2UR_CALL(
       Platform->ZeMutableCmdListExt.zexCommandListUpdateMutableCommandsExp,
-      (CommandBuffer->ZeComputeCommandListTranslated, &MutableCommandDesc));
```

Comment:
You still need the translated compute command list handle. Any call to a n extension function pointer requires all handles used are translated.